src/functions.php

```diff
@@ -272,6 +272,38 @@ function info(Configuration $config = null)
     function bin()
     {
         return function () {
+            if (!isset($_SERVER['PSYSH_IGNORE_ENV']) || !$_SERVER['PSYSH_IGNORE_ENV']) {
```

Comment:
This (and the helpful error messages) is a great call.

Reply:
Borrowed the idea from php-cs-fixer, actually. ;)

Reply:
The idea of the env variable that is, not the error messages. :P